addons/stripper/maps/surf_mesa.cfg

```diff
@@ -137,3 +137,8 @@ add:
   "classname" "env_laser"
   "hammerid" "3875"
 }
+
+filter:
```

Comment:
Just nitpicking, you can add this filter to the top of the file after the env_sun removal for cleanliness.

Like so:
```
filter:
{
  "classname" "env_sun"
}
{
  "classname" "env_soundscape"
}
```